advanced/wallets/react-wallet-v2/src/lib/smart-accounts/SafeSmartAccountLib.ts

```diff
@@ -25,4 +37,132 @@ export class SafeSmartAccountLib extends SmartAccountLib {
       }
     }
   }
+
+  async sendTransaction({ to, value, data }: { to: Address; value: bigint; data: Hex }) {
+    if (!this.client || !this.client.account) {
+      throw new Error('Client not initialized')
+    }
+    const accountDeployed = await isSmartAccountDeployed(
+      this.publicClient,
+      this.client.account.address
+    )
+    /**
+     * this is just a temporary fix for safe7579 modular account
+     * is safe7579Account is not depoyed then need to create
+     * one useroperation which first deploy and setup the account ,
+     * second user operation will be used to call the desired actions
+     * */
+    if (!accountDeployed) {
+      const setUpUserOp = await this.client.prepareUserOperationRequest({
+        userOperation: {
+          callData: await this.client.account.encodeCallData({ to, value, data })
+        },
+        account: this.client.account
+      })
+      const newSignature = await this.client.account.signUserOperation(setUpUserOp)
+
+      setUpUserOp.signature = newSignature
+
+      const setUpUserOpHash = await this.bundlerClient.sendUserOperation({
+        userOperation: setUpUserOp
+      })
+      const txHash = await this.bundlerClient.waitForUserOperationReceipt({
+        hash: setUpUserOpHash
+      })
+    }
+
+    const txResult = await this.client.sendTransaction({
+      to,
+      value,
+      data,
+      account: this.client.account,
+      chain: this.chain
+    })
+
+    return txResult
```

Comment:
Is it possible to bundle this together in a single tx?

Reply:
I will test bundling it together, 
reason behind separate calls is 
First call is calling setUpSafe method on the account. ( Entrypoint directly calls setUpSafe on account)
Second call is calling execute method on the account which then execute requested calls. ( Entrypoint directly calls execute on account) 

Possible tryout is encoding setUpSafe method to be called by account via execute method.

Reply:
@KannuSingh Alright. Lets test and make a change if possible